tests/playbooks/tests_bond_deprecated.yml

```diff
@@ -0,0 +1,96 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- hosts: all
+  vars:
+    controller_profile: bond0
+    controller_device: nm-bond
+    slave1_profile: bond0.0
+    dhcp_interface1: test1
+    slave2_profile: bond0.1
+    dhcp_interface2: test2
+  tasks:
+    - name: "INIT Prepare setup"
+      debug:
+        msg: "##################################################"
+    - import_tasks: tasks/create_test_interfaces_with_dhcp.yml
+    - import_tasks: tasks/assert_device_present.yml
+      vars:
+        interface: "{{ dhcp_interface1 }}"
+    - import_tasks: tasks/assert_device_present.yml
+      vars:
+        interface: "{{ dhcp_interface2 }}"
+    - block:
+        - name: "TEST Add Bond with 2 slaves using deprecated master"
```

Comment:
IMHO it is not clear from the name what is deprecated/tested here.

Reply:
I've clarified it. Thanks!